GetNOTAM.py

```
@@ -13,15 +13,18 @@
 #getNotam: takes the lat, long, start and end time and the page number then runs an API call to the FAA for a json of the api
 #@returns parsed_req: the json of the request
 def getNotam(effectiveStartDate, effectiveEndDate, longitude, latitude, pageNum, radius):
-  # Convert datetime strings to the desired format
-  effectiveStartDate = datetime.strptime(effectiveStartDate, "%Y-%m-%d %H:%M:%S").strftime("%Y-%m-%dT%H:%M:%SZ")
-  effectiveEndDate = datetime.strptime(effectiveEndDate, "%Y-%m-%d %H:%M:%S").strftime("%Y-%m-%dT%H:%M:%SZ")  
+  # Convert start and end dates to the desired format if provided
+  if effectiveStartDate:

```

if `not effectiveStartDate` what will happen?  does the API call below still work?  if not, we should raise an Exception of some kind to indicate the function was called with invalid parameter values

---

The API call works without a startDate or endDate so if not the program runs as normal. If a startDate and endDate are inputted then they need to be converted to the required format.